### DISCUSSION TITLE
Copilot chat has not been working all day today.!

### DISCUSSION THREAD

**Discussion by sanggilyu** [community/community#112112] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

to. GitHub staffs.
Why is GitHub not repairing Copilot chat and is it operating normally?
Copilot Chat is not working at this moment.

**Comment by JeffreyBai** [comment]
The bad news is that even Copilot itself is down as well, Total blackout...

**Reply by davecheney** [reply]
@JeffreyBai I see no requests for a copilot token from your account.  Is it possible that your employer, internet service provider, or government, may be intercepting your SSL traffic?

**Reply by davecheney** [reply]
@sanggilyu i see the same situation for you.
